### DISCUSSION TITLE
I am locked out of my account

### DISCUSSION THREAD

**Discussion by LetMeIntoMyOldAccount** [community/community#118756] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am completely locked out of my account. I cannot find any recovery codes and Github refuses to send me a recovery email. I have checked my spam folder about a dozen times. No such email was sent.

I want to talk to someone who works there. If I do not get my account back, how am I supposed to continue with my repos? I'm logged in on Github desktop, so can I recover my account there?

This is completely unacceptable and I am extremely angry. I need my account back. I need 2FA disabled for it immediately. I will do everything required of me. Just get me in touch with a real person who can help me.

**Comment by 1112779648** [comment]
ستارميكرID13288422161

**Reply by LetMeIntoMyOldAccount** [reply]
what is this, a bot?
